app/server/appsmith-server/src/main/java/com/appsmith/server/configurations/CommonConfig.java

```diff
@@ -130,4 +136,14 @@ public void setSignupDisabled(@Value("${signup.disabled}") String value) {
     public String getRtsBaseUrl() {
         return "http://127.0.0.1:" + rtsPort;
     }
+
+    public Boolean isMongoUptoDate() {
+        ComparableVersion minSupportedVersion = new ComparableVersion(minSupportedMongoVersion);
+        ComparableVersion connectedMongoVersion = new ComparableVersion(mongoDBVersion);
+        return minSupportedVersion.compareTo(connectedMongoVersion) <= 0;
+    }
+
+    public Boolean isConnectedMongoVersionAvailable() {
```

Comment:
This never returns `null`, so let's use the unboxed `boolean`.

```suggestion
    public boolean isConnectedMongoVersionAvailable() {
```